Resources/Audio/Effects/Footsteps/attributions.yml

```diff
@@ -25,3 +25,11 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from https://github.com/tgstation/tgstation"
   source: "https://github.com/nero1024/tgstation/blob/83ccc939a20489de8ab81cb47e6f8e84c490adc2/sound/effects/footstep/slime1.ogg"
+
+- files:
+  - snake1.ogg
+  - snake2.ogg
+  - snake3.ogg
+  license: "CC-BY-SA-3.0"
```

Comment:
The site you took sounds from doesn’t say that these sounds are CC-BY-SA-3.0

Reply:
It says they are completely free. So I have to delete the license?

Reply:
Free is not CC-BY-SA-3.0 which is why freesound.org is easier.

Reply:
It says on their website that they only require that their website be listed as the source. Which is accomplished here.

Reply:
Just go with "Custom" and add the stipulation.